### ISSUE TITLE
common.js location issue

### ISSUE THREAD

**Issue by factoryal** [Mrminfive/hexo-theme-skapp#37]
![image](https://user-images.githubusercontent.com/22570054/35425649-ac5503e2-029f-11e8-8d69-f77b5af4d69f.png)

I'm testing appling this theme at http://localhost:4000/mk/
Every file loads okay except common.js
Only this file is trying to load in root directory, not /mk/ directory.
How can I fix that?

Thanks for your answer and sorry for my english...

**Comment by BruceYuj** [comment]
use order hexo s to test( not hexo s -debug)

**Comment by factoryal** [comment]
@BruceYuj I tried but not working also.

**Comment by BruceYuj** [comment]
can you show me the hexo console log ?

**Comment by factoryal** [comment]
@BruceYuj 

![image](https://user-images.githubusercontent.com/22570054/35426031-56dbba9c-02a3-11e8-890d-289de6f5e6ea.png)

Is this right?

**Comment by BruceYuj** [comment]
it looks correct. common. js is concat by other js files.

**Comment by BruceYuj** [comment]
common. js has been created. Can you flush your browser and show me the error hexo log

**Comment by factoryal** [comment]
@BruceYuj 
![image](https://user-images.githubusercontent.com/22570054/35426196-7ac92646-02a4-11e8-9c1c-17f870b56099.png)

![image](https://user-images.githubusercontent.com/22570054/35426215-978f71e0-02a4-11e8-91c4-9e0d29d0de9a.png)

I force removed the cache and loaded again. but nothing changes.

**Comment by BruceYuj** [comment]
what is your OS

**Comment by factoryal** [comment]
Windows 10 Pro Insider Preview Build 17074.rs_prerelease_fit.180116-1539
![image](https://user-images.githubusercontent.com/22570054/35426265-0a375708-02a5-11e8-8272-b5bf02133675.png)

**Comment by BruceYuj** [comment]
@fatoryal because i can't use my computer now,  i can't test your problem.  can you wait for some hours or check your configuration or restart your computer.

**Comment by BruceYuj** [comment]
@factoryal  
this is my test `hexo s` log
![hexo s](https://user-images.githubusercontent.com/19621739/35427333-a6779c1a-02a3-11e8-8b28-d72cd1b4187e.PNG)

this is the post site `http://localhost:4000/2018/01/22/2017-1-22-context-and-executionContext/`
dev html structure. 
![hexo-post](https://user-images.githubusercontent.com/19621739/35427344-b99de40c-02a3-11e8-9be6-e4996dbdffac.PNG)

When I use order `hexo g`, I can find the `common.js` file in the `public\js` folder.
![default](https://user-images.githubusercontent.com/19621739/35427421-41ab83fe-02a4-11e8-924e-bef3b2a5a1e0.PNG)

**Comment by factoryal** [comment]
@BruceYuj Yeah, It works when I set server location as root (localhost:4000/)
But It is not working when server location at subdirectory (localhost:4000/mk/)
Directory of js files are absolute not relative.

**Comment by BruceYuj** [comment]
@factoryal  That is because the index page does not need the common.js file.

**Comment by BruceYuj** [comment]
@factoryal  Hi, do you fix your problem?

**Comment by factoryal** [comment]
Sorry for late reply.
I didn't fix the problem yet.
Some other Hexo themes causes this problem also. (example: https://github.com/stkevintan/hexo-theme-material-flow/issues/4)
Now, I'm working in root directory.
